Docs/User Guide.md

```diff
@@ -39,6 +39,14 @@ The following table describes the available endpoint configurations.
 
 _NOTE: Configure the SLSC Switch Custom Device before configuring the Routing and Faulting Custom Device._
 
+### Reservation Group
```

Comment:
@WillCawley please review this section.

Reply:
Here are my suggestions. Feel free to take or disregard as needed:

The SLSC Switch Custom Device initializes a SLSC Switch session with a custom _reservation group_. This initialization allows sessions in the same group to concurrently reserve the device. Avoid modifying SLSC Switch session properties through these external sessions, as this will corrupt the switching state and establish unintended connections.

Use SLSC Switch 20.0 or newer to apply the reservation group. If you use SLSC Switch 19.x, the device will silently ignore the reservation group.

By default, the reservation group for the SLSC Switch Custom Device is the path to the module in the system definition file. Use the scripting API to update this path.